docs/integrations/app-integrations/youtube.mdx

```diff
@@ -5,66 +5,81 @@ sidebarTitle: YouTube
 
 In this section, we present how to connect YouTube to MindsDB.
 
-[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users can upload, view, share, and interact with videos created by individuals and organizations from around the world.
+[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users
+can upload, view, share, and interact with videos created by individuals and organizations from around the world.
 
-Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments, detect sentiment of comments, or use AI models to create replies.
+Comment Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments,
+detect sentiment of comments, or use AI models to create replies.
 
 ## Connection
 
-This handler is implemented using the `google-api-python-client` library, a Python library that from where we can use YouTube v3 API.
+First, we have to install the required python packages.
+```bash
+pip install mindsdb[youtube]
+```
+
+## Usage
+
+This handler is implemented using the ```google-api-python-client``` library, a Python library that from where we can use YouTube v3 API.
 
 The only required argument to establish a connection are as follows:
 
-* `youtube_api_token` which is a Google API key used for authentication.
+* ```youtube_api_token``` which is a Google API key used for authentication.
 
 <Tip>
-Check out [this guide](https://blog.hubspot.com/website/how-to-get-youtube-api-key) on how to create the API key in order to access YouTube data.
+Check out [this guide](https://blog.hubspot.com/website/how-to-get-youtube-api-key) on how to create the API key in
+    order to access YouTube data.
 </Tip>
 
-The first step is to create a database with the new `Youtube` engine using the following command:
+The first step is to create a database with the new ```youtube``` engine using the following command:
```

Comment:
The inline code uses single backticks. So it should be "...with the new \`youtube` engine..."